src/libs/TransactionUtils.ts

```diff
@@ -913,6 +927,37 @@ function getTransactionID(threadReportID: string): string {
     return IOUTransactionID;
 }
 
+function buildNewTransactionAfterReviewingDuplicates(reviewDuplicateTransaction: OnyxEntry<ReviewDuplicates>): Partial<Transaction> {
+    const originalTransaction = allTransactions?.[`${ONYXKEYS.COLLECTION.TRANSACTION}${reviewDuplicateTransaction?.transactionID}`] ?? undefined;
+    const {duplicates, ...restReviewDuplicateTransaction} = reviewDuplicateTransaction ?? {};
+
+    return {
+        ...originalTransaction,
+        ...restReviewDuplicateTransaction,
+        modifiedMerchant: reviewDuplicateTransaction?.merchant,
+        merchant: reviewDuplicateTransaction?.merchant,
+        comment: {comment: reviewDuplicateTransaction?.description},
```

Comment:
Omitting other `reviewDuplicateTransaction.comment` properties caused an issue where if default tax is selected, `Default` label is not shown next to tax name